README.md

```diff
@@ -5,3 +5,7 @@ This is a collection of miscellaneous tools released by the BloodHound team. See
 * DBCreator - Tool to generate randomized Neo4j databases for use with BloodHound
 * BloodHoundAnalytics.pbix - Proof of concept charting capability
 * BloodHoundAnalytics.py - Proof of concept audit script
+
```

Comment:
Should the proposed build changes get merged in a separate PR you are more than welcome to leave yourself and attribution 😄. But let's remove it for this one.